### PR TITLE
Included a getToken method call in the Equity object initialization.

### PR DESCRIPTION
**Pull request by cryarchy**
The goal is to avoid having the user having to explicitly call the initial *getToken* method and then wrap the application's logic within a callback to the method.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,13 +26,16 @@ const Equity = require('equityjs');
 
 const equity = new Equity({
   consumerKey: 'YOUR_CONSUMER_KEY',
-  consumerSecret: 'YOUR_CONSUMER_SECRET'
+  consumerSecret: 'YOUR_CONSUMER_SECRET',
+  username: 'user', //(optional)
+  password: 'pass', //(optional)
+  grant_type: 'password' //(optional)
```
Comment (GochoMugo) [inline README.md:32]: By passing in `username` and `password`, we are forcing the developer to create multiple instances of `Equity` for each user their application handles. That is inefficient and cumbersome. I believe there's a better way to handle this.

File: README.md [hunk 2]
```diff
@@ -26,13 +26,16 @@ const Equity = require('equityjs');
...
 );
 ```
 
-Both are **required** parameters. You can obtain your keys from [here](https://developers.equitybankgroup.com/user/me/apps), by creating an app.
+You can obtain your *consumerKey* and *consumerSecret* keys from [here](https://developers.equitybankgroup.com/user/me/apps), by creating an app.
 
-Refer to [examples.js](https://github.com/kamikazechaser/EquityJs/blob/master/examples.js) for more usage indformation.
+Refer to [examples.js](https://github.com/cryarchy/EquityJs/blob/master/examples.js) for more usage information.
```
Comment (GochoMugo) [inline README.md:38]: Typo in the URL?

File: examples.js [hunk 3]
```diff
@@ -36,11 +39,23 @@ equity.getToken({
...
+equity.createPayment({
+    mobileNumber: "0728313140",
+    amount: "1000",
+    description: "payment",
+    type: "payment",
+    auditNumber: "2423424ASAD"
+  }, function(error, response) {
+    // Handle The Response Here
+  });
+
```
Comment (GochoMugo) [inline examples.js]: too much personal details, don't you think?

File: README.md (hunk 1 above)
Comment (cryarchy) [reply README.md:32]: Cool I'll see if I can figure out how to implement it as such. I hadn't thought of it in that context.

File: README.md (hunk 2 above)
Comment (cryarchy) [reply README.md:38]: Yeah! + I updated the example to reflect the new usage.

File: examples.js (hunk 3 above)
Comment (cryarchy) [inline examples.js]: ?

File: examples.js [hunk 4]
```diff
@@ -36,11 +39,23 @@ equity.getToken({
 });
 }, refreshToken);
 
+// Create a Payment
+equity.createPayment({
+    mobileNumber: "0728313140",
```
Comment (GochoMugo) [inline examples.js:44]: too much personal details, don't you think?

**Comment by kamikazechaser** [comment]
Outdated by release v2.0.0

**Pull request closed without merging**